# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/WebGLParent.h
# Commit: 16c5dd2cfd60
# Full Hash: 16c5dd2cfd600a62c80ca17cfb35c0868e016056
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-15 21:48:38
# Regressor Bug: 1632249
# File Overlap Count: 1
# Description:
#   Bug 1632249 - Support compositing out-of-process WebGL. r=handyman,lsalzman,nical,geckoview-reviewers,agi,imanol
#   
#   * Majorly simplity CanvasRenderer
#   * Replace GLScreenBuffer with trivial GLSwapChain
#   * Use descriptor structs so that future SharedSurface changes aren't so painful
# ==============================================================================

diff -r ca8139d58d58 -r 16c5dd2cfd60 dom/canvas/WebGLParent.h
--- a/dom/canvas/WebGLParent.h	Fri Jun 12 21:53:14 2020 +0000
+++ b/dom/canvas/WebGLParent.h	Mon Jun 15 18:25:55 2020 +0000
@@ -37,13 +37,10 @@
       const webgl::InitContextDesc&, UniquePtr<HostWebGLCommandSinkP>&& aSinkP,
       UniquePtr<HostWebGLCommandSinkI>&& aSinkI, webgl::InitContextResult* out);
 
-  RefPtr<layers::SharedSurfaceTextureClient> GetVRFrame(webgl::ObjectId);
-
   // Drain the command queue now.  Used by synchronous IpdlQueue consumers.
   bool RunQueue(uint64_t) { return RunCommandQueue(); }
 
-  // For IPDL:
-  WebGLParent();
+  WebGLParent();  // For IPDL
 
  private:
   ~WebGLParent();